examples/federated-learning/tff/nvflare/base-deployment/base-deployment.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nvflare
+  namespace: nvflare
```

Comment:
Don't hardcode the namespace. Set it with kustomize. Check the rest of the resources as well.

Reply:
Is it better ?